contracts/Authorizable.sol

```diff
@@ -4,18 +4,23 @@ pragma solidity >=0.4.22 <0.9.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract Authorizable is Ownable {
-    address public trustee;
+    address[] public trustees;
```

Comment:
Need to be able to set max length of trustee list.

Could be a private function that would call from FFV3 contract consturctor

Reply:
Changed array to map